File: 2274-keep-multiplying-found-values-by-two/keep-multiplying-found-values-by-two.java
class Solution {
    public int findFinalValue(int[] nums, int original) {
        if(!search(nums,original)){
            return original;
        }
        int s=0;
        int o=original;
        while(search(nums,o)){
            o=o*2;
        }
        return o;
    }
    public static boolean search(int num[],int target){
        for(int i:num){
            if(i==target){
                return true;
            }
        }
        return false;
    }
}

File: 2274-keep-multiplying-found-values-by-two/keep-multiplying-found-values-by-two.py
class Solution(object):
    def findFinalValue(self, nums, original):
        """
        :type nums: List[int]
        :type original: int
        :rtype: int
        """
        def search(nums,target):
            for i in nums:
                if i==target:
                    return True
            return False

        while(search(nums,original)):
            original*=2
        return original       